### ISSUE TITLE
Health roadmap

### ISSUE THREAD

**Issue by stilnat** [RE-SS3D/SS3D#1192]
## Summary

Health roadmap based upon our health design : https://ss3d.gitbook.io/design/entities/health/body-systems
 
- First iteration, focused on detachable body parts and basic damages mechanics to body parts.  
#1247

- Second iteration, focused on the circulatory system. 
#1248 

- Third iteration, focused on animating part 1 and 2
#1283

**Comment by TaylorNAlbarnaz** [comment]
How will the brain take damage in the first milestone? It's not clear from the points shown, if the heart is added in the second milestone, does that mean the player could in theory survive being a severed head? since in theory the brain itself wouldn't have taken damage, and since there would be no blood pumping mechanic from the heart it wouldn't die from suffocation damage

I'm ok with it not taking any indirect damage until the next milestones, i just want it to be more clear

**Comment by stilnat** [comment]
@TaylorNAlbarnaz yes, it's precisely what it implies, you can severe a head without killing a player following this milestone. It only takes direct damages for now.
